mpfs_icicle: samples/drivers/flash_shell: Build failed in CI

**Describe the bug**
Testing flah shell sample in CI fails on mpfs_icicle target
```
twister -p mpfs_icicle -T  samples/drivers/flash_shell/
```
**To Reproduce**
Steps to reproduce the behavior:
1. mkdir build; cd build
2. cmake -DBOARD=board\_xyz
3. make
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Impact**
CI blocker. See https://github.com/zephyrproject-rtos/zephyr/pull/74075

**Logs and console output**
```
[...] zephyrproject/zephyr/drivers/flash/spi_nor.c [...]
zephyrproject/zephyr/include/zephyr/toolchain/gcc.h:87:36: error: static assertion failed: "jedec,spi-nor jedec-id required for non-runtime SFDP"
   87 | #define BUILD_ASSERT(EXPR, MSG...) _Static_assert(EXPR, "" MSG)
      |                                    ^~~~~~~~~~~~~~

```

--- a/boards/microchip/mpfs_icicle/mpfs_icicle.yaml
+++ b/boards/microchip/mpfs_icicle/mpfs_icicle.yaml
@@ -9,4 +9,5 @@ testing:
   ignore_tags:
     - net
     - bluetooth
+    - flash
 vendor: microchip

--- a/boards/microchip/mpfs_icicle/mpfs_icicle_smp.yaml
+++ b/boards/microchip/mpfs_icicle/mpfs_icicle_smp.yaml
@@ -9,4 +9,5 @@ testing:
   ignore_tags:
     - net
     - bluetooth
+    - flash
 vendor: microchip

